playbooks/common/openshift-cluster/upgrades/rpm_upgrade.yml

```diff
@@ -7,7 +7,7 @@
 # core team tells us not to rely on yum module transaction flattening anyway.
 
 - name: Upgrade master packages
-  package: name={{ master_pkgs | reject('equalto', '') | join(',') }} state=present
+  package: name={{ master_pkgs | join(',') | replace(',,', ',') }} state=present
   vars:
     master_pkgs:
       - "{{ openshift.common.service_type }}{{ openshift_pkg_version }}"
```

Comment:
For -sdn-ovs below, we can just use openshift.common.service_type ~ openshift_pkg_version instead of '' and then we can avoid this ugly logic.  Specifying the same package name twice should work, I believe.

Reply:
So then `openshift.common.service_type ~ openshift_pkg_version` would appear twice in the list? Okay, that might work